libcst/codegen/gather.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
github is not smart enough to mark `libcst/codegen/gather.py` and `libcst/codegen/gen_type_mapping.py` as moved changes, that can make code review much easier.